gramjs/tl/api.d.ts

```diff
@@ -5088,15 +5088,15 @@ export namespace Api {
     }
     export class ReceivedNotifyMessage extends VirtualClass<{
         id: int;
-        // flags: int;
+        flags: int;
```

Comment:
The type of flags here is `flags:int`, which is not a flag indicator (`flags:#`).
By changing `=== 'flags'` to `.flagIndicator`
 in the last commit accidently fixed this :)

Reply:
But the `flags` here is "Reserved for future use", which is useless now... If user doesn't provided the `flags`, will this cause an Error?

Reply:
this is a result from telegram so the user will not be sending it anyway. we can ignore it for now.